concent_api/core/payments/bankster.py

```diff
@@ -0,0 +1,47 @@
+from typing import Tuple
+
+from django.conf import settings
+
+from common.constants import ConcentUseCase
+from core.constants import ETHEREUM_ADDRESS_LENGTH
+from core.payments import service
+
+
+def claim_deposit(
+    concent_use_case: ConcentUseCase,
+    requestor_ethereum_address: str,
+    provider_ethereum_address: str,
+    subtask_cost: int,
+) -> Tuple[bool, bool]:
+    """
+    The purpose of this operation is to check whether the clients participating in a use case have enough funds in their
+    deposits to cover all the costs associated with the use case in the pessimistic scenario.
+    """
+
+    assert isinstance(concent_use_case, ConcentUseCase)
+    assert isinstance(requestor_ethereum_address, str)
+    assert isinstance(provider_ethereum_address, str)
+    assert isinstance(subtask_cost, int) and subtask_cost > 0
```

Comment:
are we sure that `subtask_cost` cannot be  `0`?

Reply:
That is a tricky one:
1. `TaskToCompute` with 0 value can be created, serialized, deserialized etc.
2. In `validate_task_to_compute` we call a function `validate_positive_integer_value` for `price`. Inside we check if `value < 0`. **Should be opposite**.
3. In issue `subtask_cost` is described as `Must be greater than zero.`.

I think we should add this point to our discussion on Wednesday @kbeker .

Or ask Golem team @PaweuB .

Reply:
ok, I approve but please refrain from merging until it's solved